integrations/telegram/src/index.ts

```diff
@@ -208,7 +209,6 @@ const integration = new bp.Integration({
     await client.createMessage({
       tags: {
         [idTag]: messageId.toString(),
-        [fromUserIdTag]: userId.toString(),
```

Comment:
Removed the telegram user ID from the message tags as it's already in the conversation tags and storing it in the message tags is too redundant.